src/Calculator.jsx

```diff
@@ -10,11 +10,11 @@ function Calculator() {
   }, [operand]);
 
   const cubed = useMemo(() => {
-    return operand ** 2;
+    return operand * operand * operand;
```

Comment:
Is this the best way to solve this? What if instead of cubed you had to multiply by power of 4? This approach doesn't offer much flexibility.